autorest/codegen/__init__.py

```diff
@@ -289,6 +306,9 @@ def _build_code_model_options(self) -> Dict[str, Any]:
             "low_level_client": low_level_client,
             "combine_operation_files": self._autorestapi.get_boolean_value("combine-operation-files", version_tolerant),
             "python3_only": python3_only,
+            "package_mode": self._autorestapi.get_value("package-mode"),
+            "package_pprint_name": self._autorestapi.get_value("package-pprint-name"),
+            "package_configuration": self._autorestapi.get_value("package-configuration")
```

Comment:
since we're exposing `package-configuration` too can we have tests for it?

Reply:
here is the test: https://github.com/Azure/autorest.python/blob/47e131b686a33bdfebb8fd64f13d2605731e3c1f/test/azure/legacy/specification/packagemodecustomize/README.md